Index: app/src/main/java/com/example/hector/mercadolibre/retrofit/PaymentService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/hector/mercadolibre/retrofit/PaymentService.java	(date 1528091812196)
+++ app/src/main/java/com/example/hector/mercadolibre/retrofit/PaymentService.java	(date 1528091812196)
@@ -0,0 +1,24 @@
+package com.example.hector.mercadolibre.retrofit;
+
+import com.example.hector.mercadolibre.models.CardIssuers;
+import com.example.hector.mercadolibre.models.Installment;
+import com.example.hector.mercadolibre.models.PaymentMethod;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.http.Field;
+import retrofit2.http.FormUrlEncoded;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface PaymentService {
+    @GET("payment_methods")
+    Call<List<PaymentMethod>> getPaymentMethods(@Query("public_key") String publicKey);
+
+    @GET("payment_methods/card_issuers")
+    Call<List<CardIssuers>> getCardIssuers(@Query("public_key") String publicKey, @Query("payment_method_id") String paymentMethodId);
+
+    @GET("payment_methods/installments")
+    Call<List<Installment>> getInstallment(@Query("public_key") String publicKey, @Query("payment_method_id") String paymentMethodId, @Query("issuer.id") String issuerId);
+}
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/vcs.xml	(date 1528092369449)
+++ .idea/vcs.xml	(date 1528092369449)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
